h/panels/navbar.py

```diff
@@ -29,14 +30,20 @@ def navbar(context, request, search=None, opts=None):
                 'title': group.name,
                 'link': request.route_url('group_read', pubid=group.pubid, slug=group.slug)
             })
-            groups_suggestions.append({
-                'name': group.name,
-                'pubid': group.pubid
-            })
         user_activity_url = request.route_url('activity.user_search',
                                               username=request.user.username)
         username = request.user.username
 
+    # make all groups visible in the search auto complete
+    groups = profile_groups_factory(None, request).all(request.user, request.authority)
+    for group in groups:
+        # don't return __world__ group
+        if group['id'] == '__world__':
```

Comment:
I'm not sure that we should exclude the Public group from this list, but let's discuss that once the autosuggest stuff has been extracted into a separate PR.